templates/login/login_core.html.twig

```diff
@@ -203,23 +206,23 @@
 
     // Click-handler for signin button
     function do_google_signin() {
-        google_signin = true;
+       google_signin = true;
     }
 
     // When Google sign-in successful, sign in to the app, but only
     // if the button was clicked (otherwise we would automatically login)
     function onSignInSuccess(googleUser) {
-        if (google_signin === true) {
-            const auth_response = googleUser.getAuthResponse();
-            const id_token = auth_response.id_token;
+        
+    //        const auth_response = googleUser.getAuthResponse();
```

Comment:
ok to remove this instead of commenting

Reply:
@stephenwaite I made changes like you said. Could you review 
commit cd882f845a5caab6d61894e474603a5dbd362171  ?
What is the next step?
Thanks